nucleus/model_run.py

```diff
@@ -127,6 +127,13 @@ def predict(
             "predictions_ignored": int,
         }
         """
+        # TEMP Workaround to avoid annotation_id conflicts across distinct model_runs
```

Comment:
```suggestion
        # TEMP Workaround to avoid annotation_id conflicts across distinct model_runs
        # Revert once this is implemented: https://app.shortcut.com/scaleai/story/344585/relax-nucleus-model-prediction-on-conflict-condition-to-annotation-id-dataset-item-id-model-run-id
```